ide/docker/Dockerfile

```diff
@@ -16,3 +16,10 @@ WORKDIR /opt/xcsoar
 
 COPY ./docker/bin/* /usr/local/bin/
 RUN cd /usr/local/bin && chmod 755 ./*
+
+# Environment to support running XCSoar
+ENV DISPLAY=host.docker.internal:0
```

Comment:
Hi, 
first of all thank you for following my call to action. :) +1 

Is this safe to hardcode for all possible runtime environments?

I saw for example this approach for a linux host: (the osx host will still require the above method)
https://medium.com/@SaravSun/running-gui-applications-inside-docker-containers-83d65c0db110

Reply:
I already tried this. The article you mention is targeted towards Linux. I was unable to get this to run within reasonable time on macOS. The fact, that Docker runs within a virtual machine on macOS complicates things. The `DISPLAY` variable holds the path to a UNIX socket when using XQuartz.

Now that I am thinking about this again, the solution I provided here is only really suitable for macOS and also only because the OSX target is currently not working. (PC target is working I guess.)

Maybe it is not worth to put this all into XCSoar just to work around a defunct OSX target. Better to fix the OSX target or to use my buildenv as an alternative as long as it is defunct.